spec/install/gemfile/sources_spec.rb

```diff
@@ -588,26 +569,6 @@
           gem "rack-obama"
         G
 
-        # It creates this lockfile.
```

Comment:
unrelated change?

Reply:
Yeah, I can extract this changes to a separate PR. :+1:

Reply:
Extracted to #7167. Will rebase this PR once that one is merged, so that these changes disappear from the diff.